src/app/page/Home/Home.js

```diff
@@ -56,6 +59,67 @@ const useStyles = makeStyles((theme) => ({
     color: "white",
     fontSize: "24px",
   },
+  OrgBackgroundImg: {
+    backgroundImage: `url(${SplashImage1})`,
+    backgroundSize: "100% 100%",
+    height: "450px"
+  },
+  OrgDetails: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1)
+  },
+  NeedHelpSection: {
+    backgroundImage: `url(${HomeImage3})`,
+    backgroundSize: "100% 100%",
+    height: "240px",
+    display: "flex"
+  },
+  WhiteBgText: {
+    background: "rgb(255, 255, 255, 0.8)",
+    padding: `${theme.spacing(1)}px ${theme.spacing(2)}px`,
+    margin: "auto"
+  },
+  BgBlack: {
+    backgroundColor: "black"
+  },
+  WhiteBgTxt: {
+    padding: "10px",
+    background: "rgb(255, 255, 255, 0.8)"
+  },
+  WantToHelpSection: {
+    backgroundImage: `url(${HomeImage4})`,
+    backgroundSize: "100% 100%",
+    height: "240px",
+    display: "flex"
+  },
+  Emoticon: {
+    height: "36px",
+    width: "36px",
+    marginLeft: theme.spacing(1),
+    verticalAlign: "middle"
+  },
+  PurpleBg: {
+    backgroundColor: theme.color.deepPurple
```

Comment:
Use of deepPurple doesn't seem to exist

Reply:
its actually red now, I dont know, the theme keep changing from the designer, lets add deepPurple back ....
but it is red.

Reply:
Can we use less specific variables for these? like backgroundPrimary or backgroundLight or something that isn't tied to a color? So if Design decides that these colors change, we just update that variable at the theme level, and changes like this wouldn't be affected

Reply:
I would like that as well, but it is the design in Figma that show up as color red/deepPurple instead. We should ask this question to the designer

Reply:
Can we stick to the colors provided by [material-ui library](https://material-ui.com/customization/color/)?